components/brave_shields/browser/tracking_protection_service_browsertest.cc

```diff
@@ -179,7 +180,7 @@ IN_PROC_BROWSER_TEST_F(TrackingProtectionServiceTest, StorageTrackingBlocked) {
   GURL tracking_url = embedded_test_server()->GetURL("tracker.com",
     kStoragePage);
 
-  GURL url = embedded_test_server()->GetURL(std::string("/server-redirect?") +
+  GURL url = embedded_test_server()->GetURL(std::string(kRedirectPage) +
```

Comment:
nit: actually you don't need to explicitly call `std::string` constructor, just `->GetURL(kRedirectPage)`